<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>955</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font-size: 18px</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Operation with Fraction &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>701</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Enter Input</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Numerator: </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditNumerator">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>20</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Denominator: </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditDenominator">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>30</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>701</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Operation</string>
    </property>
    <widget class="QPushButton" name="pushButtonAddition">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Addition</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>40</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Substraction</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonMultiplication">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>40</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Multiplication</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonDivision">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>40</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Division</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditResult">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>641</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>701</width>
      <height>411</height>
     </rect>
    </property>
    <property name="title">
     <string>Display Fraction</string>
    </property>
    <widget class="QLineEdit" name="lineEditAllFraction">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>40</y>
       <width>661</width>
       <height>131</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidgetListFraction">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>661</width>
       <height>192</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>STT</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Numerator</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Denominator</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>701</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Features</string>
    </property>
    <widget class="QPushButton" name="pushButtonCreate">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Create</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonReadAll">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>50</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Read all</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonUpdate">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>50</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Update</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonDelete">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>50</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>725</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
